src/main/java/com/google/sps/servlets/FetchAnswersServlet.java

```diff
@@ -35,15 +38,90 @@
 /** 
  * Retrieves the answers to be displayed on the page.
  */
-@WebServlet("/answers")
+@WebServlet("/fetch-answers")
 public class FetchAnswersServlet extends HttpServlet {
 
   /** 
-   * Gets the answers for a single question and send them back as JSON.
-   * TODO(shaargtz): Modify to get style and functionality right.
+   * This method will get the answers for a single question and send them back as JSON.
    */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // ID of the question to which the answers correspond.
+    int question_id = Integer.parseInt(request.getParameter("id"));
 
+    // Create a map that will hold all of the answers from the query.
+    // Each <int> will be an answer's id, and will be used to avoid creating duplicate
+    // answers and easily add a <Comment> to the correspondind <Answer>.
+    Map<Integer, Answer> answers = new HashMap<>();
+
+    String query = Utility.fetchAnswersAndCommentsQuery;
```

Comment:
Why did you make this a Utility variable? Is it used in other files?

Reply:
i find it more readable since it is a very large string, and having the sql constants there is no confusion on what the rest of the stuff does